pkg/pillar/cmd/volumemgr/updatestatus.go

```diff
@@ -274,18 +274,21 @@ func doUpdateContentTree(ctx *volumemgrContext, status *types.ContentTreeStatus)
 		// for now, we only load containers into containerd
 		// TODO: next stage, load disk images as well
 		if status.Format == zconfig.Format_CONTAINER {
-			if err := ctx.casClient.IngestBlobsAnsCreateImage(
+			loadedBlobs, err := ctx.casClient.IngestBlobsAnsCreateImage(
```

Comment:
Can this miss anything? E.g. if I have 4 blobs (1,2,3,4). 1&2 are already loaded; 3&4 I am loading now. After this change, I _only_ loop through those returned by `IngestBlobs...()`, as opposed to a generalized `lookupBlobStatuses()`. 

Will I loop through 1,2,3,4? Or will `IngestBlobs...()` return just 3&4, because those are the ones I am loading, and thus will I miss 1&2?

I just cannot tell from reading this.

Reply:
Since we loaded only 3&4 here, we will mark only those respective BlobStatus as LOADED. Since 1&2 were already loaded, they would already have BlobStatus.state = LOADED.
There is a check in IngesBlob() where it skips the blobs with LOADED state and hence it handles and returns blobs that were not already loaded.

Reply:
So it missed them, but that is OK? OK fine. Thanks for clarifying.